src/App.js

```diff
@@ -53,13 +55,17 @@ function GetAccount({ change_auth, get_accounts, get_categories, get_payees, aut
   );
 
   const AuthPages = () => (
-    <Layout>
-      <Switch>
-        {routes.map((route, index) => (
-          <Route exact key={index} path={route.path} component={route.component} />
-        ))}
-        <Redirect to="/" />
-      </Switch>
+    <Layout style={{ minHeight: '100vh', backgroundColor: 'rgb(240, 242, 245)' }}>
```

Comment:
in react, this is not a good way to styling.

it is not recommended to use `style={{}}`

better to follow @kekakuka 's way.


- comp.js
- comp.css